carmin-server/server/resources/models/path.py

```diff
@@ -35,27 +37,22 @@ def __eq__(self, other):
         return self.__dict__ == other.__dict__
 
     @classmethod
-    def object_from_pathname(cls, platform_data_path: str,
-                             relative_path_to_resource: str):
+    def object_from_pathname(cls, absolute_path_to_resource: str):
         """object_from_pathname takes the path of the platform data root directory
         as its first argument, and the path of the requested resource
         relative to the root directory as second argument. It then returns a
         Path object based on the associated file or directory.
         """
-        absolute_path_to_resource = os.path.join(platform_data_path,
-                                                 relative_path_to_resource)
+
         is_directory = os.path.isdir(absolute_path_to_resource)
 
         # TODO: Add execution_id to Path object
```

Comment:
The instance attribute is there, but there is no way to retrieve the execution associated with the `path` since the `execution` table does not yet exist. This will have to be done once that table has been created.